### DISCUSSION TITLE
Option to switch to week view in calendar page

### DISCUSSION THREAD

**Discussion by OSTraining** [publishpress/PublishPress-Planner#1252] [Feature Requests]
This will allow more room to display the available content

┆Attachments: <a href="https:&#x2F;&#x2F;ostraining.atlassian.net&#x2F;secure&#x2F;attachment&#x2F;19102&#x2F;Calendar_%E2%80%B9_Breeze_%E2%80%94_WordPress.jpg">Calendar_‹_Breeze_—_WordPress.jpg</a>

**Comment by OSTraining** [comment]
➤ Anderson Grüdtner Martins commented:

You mean, kind of a shortcut for the 1 week filter? [~htmgarcia]

**Comment by OSTraining** [comment]
➤ Valentin Garcia commented:

[~andergmartins] yes

**Comment by OSTraining** [comment]
➤ Anderson Grüdtner Martins commented:

[~htmgarcia] Maybe another option would be to remove the checkbox and add 5 buttons, from 1 to 5 weeks? The icon could be lines according to the respective amount of weeks. What do you think?

**Comment by OSTraining** [comment]
➤ Valentin Garcia commented:

[~andergmartins] I was thinking in something simpler to display content: month / week

**Comment by andergmartins** [comment]
Related to #912
